tools/check_format.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+bazel run @envoy//tools:check_format.py -- \
+  --skip_envoy_build_rule_check  --namespace_check Nighthawk --include_dir_order envoy \
```

Comment:
Do we want to treat `absl`, `gtest`, `gmock` and `external` special in here? E.g. move them higher up?

Reply:
I prefer to have them in the final block, as with Envoy. These are all the misc. deps that don't get an explicit callout.

Reply:
I see, sgtm